README.rst

```diff
@@ -20,13 +20,13 @@ To install redis-py, simply:
 
 .. code-block:: bash
 
-    $ sudo pip install redis
+    $ pip install redis
 
 or from source:
 
 .. code-block:: bash
 
-    $ sudo python setup.py install
+    $ python setup.py install
```

Comment:
You are actually supposed to use pip for pretty much all installation.  A local install might be `pip install .` with `.` being the working directory or replaced with whatever path to the directory holding `setup.py`.  Depending how granular you like to be, this could certainly be argued to be a separate change.

Reply:
Was this being left for fixing later?